tests/harness/executor/executor.go

```diff
@@ -21,10 +21,12 @@ func main() {
 	ccFlag := flag.Bool("cc", false, "Run c++ test harness")
 	javaFlag := flag.Bool("java", false, "Run java test harness")
 	pythonFlag := flag.Bool("python", false, "Run python test harness")
+	externalHarnessFlag := flag.String("external_harness", "", "Run an external test harness through the given executable")
```

Comment:
The help text for this flag sounds like it toggles a boolean. Can you reword this to note it is the path of a binary to be executed?